apps/ledger-live-mobile/src/screens/Settings/Debug/Features/BLEDevicePairingScreen.tsx

```diff
@@ -0,0 +1,62 @@
+import React, { useCallback } from "react";
+import { Flex } from "@ledgerhq/native-ui";
+import { SafeAreaView } from "react-native-safe-area-context";
+import { Device } from "@ledgerhq/types-devices";
+
+import { ScreenName } from "../../../../const";
+import BleDevicePairingFlowComponent, {
+  SetHeaderOptionsRequest,
+} from "../../../../components/BleDevicePairingFlow";
+import { bleDevicePairingFlowHeaderOptions } from "../../../BleDevicePairingFlow";
+import { SettingsNavigatorStackParamList } from "../../../../components/RootNavigator/types/SettingsNavigator";
+import {
+  RootComposite,
+  StackNavigatorProps,
+} from "../../../../components/RootNavigator/types/helpers";
+
+export type Props = RootComposite<
+  StackNavigatorProps<SettingsNavigatorStackParamList, ScreenName.DebugBLEDevicePairing>
+>;
+
+const BLEDevicePairingScreen: React.FC<Props> = ({ navigation, route }) => {
```

Comment:
We're trying to follow a name convention where we put a capitalized letter only when changing word. Can you rename it to `BleDevicePairingScreen` pls ?
I know that for `BLEPairingFlow` i named it with `BLE` at the time :sorry: . But it should be `BlePairingFlow`. Can you rename it too pls ? With the associates ScreenName